README.md

```diff
@@ -1,51 +1,82 @@
-Lambdapi- Implementation of the λΠ-calculus modulo rewriting
-============================================================
+Lambdapi, a proof assistant based on the λΠ-calculus modulo rewriting
+=====================================================================
 
-Lambdapi is an implementation of the λΠ-calculus modulo rewriting, which
-is the system of Dedukti (https://github.com/Deducteam/Dedukti).
+[![Join the chat at https://gitter.im/dedukti3/community](https://badges.gitter.im/dedukti3/community.svg)](https://gitter.im/dedukti3/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Perhaps it needs to be updated now?

Reply:
It may be better to do the change ourselves, the diff is weird. The url for the badge is accessible from the room

Reply:
Do you mean that we should close this PR and add the badge in the README ourself?

Reply:
Yeah I think so.